examples/composer.json

```diff
@@ -17,6 +17,7 @@
         "probots-io/pinecone-php": "^1.0",
         "psr/http-factory-implementation": "*",
         "symfony/ai-agent": "@dev",
+        "symfony/ai-chat": "@dev",
```

Comment:
Either we don't need it or we need to adjust an example to use the new namespace

Reply:
It's needed for `examples/misc/persistent-chat`, the namespaces import are already adjusted.